openstack_exporter/collectors/cinderbackend.py

```diff
@@ -250,136 +273,97 @@ def collect(self):
             caps = pool['capabilities']
             backend = caps['volume_backend_name']
             shard_name = caps['vcenter-shard']
-            LOG.debug("Got stats for pool {} ({}/{})".format(
-                self.region,
-                shard_name,
-                backend))
-
             volume_type = volume_types[backend]
             data = self._parse_pool_data(pool, volume_type)
             pool_name = data['pool']
+            LOG.debug(f"Got stats for {self.region} {shard_name}/{backend}/{pool_name}")
 
-            g = GaugeMetricFamily('cinder_per_volume_gigabytes',
-                                  'Cinder max volume size in GiB',
-                                  labels=['backend', 'pool', 'shard'])
-            g.add_metric([backend, pool_name, shard_name],
-                         value=quota_obj.per_volume_gigabytes)
-            yield g
+            yield self.add_gauge_metric_gauge(
+                'cinder_per_volume_gigabytes',
+                'Cinder max volume size in Gib',
```

Comment:
`GiB` went to `Gib` here. Is that correct?

Reply:
done